graph-db/commonUtils/indexing/indexCmrCollection.js

```diff
@@ -1,12 +1,15 @@
+const gremlin = require('gremlin')
+
+const gremlinStatistics = gremlin.process.statics
 const { indexRelatedUrl } = require('./indexRelatedUrl')
 
 /**
  * Given a collection from the CMR, index it into Gremlin
  * @param {JSON} collection collection object from `items` array in cmr response
- * @param {Gremlin Traversal Object} gremlin connection to gremlin server
+ * @param {Gremlin Traversal Object} g connection to gremlin server
  * @returns
  */
-exports.indexCmrCollection = async (collection, gremlin) => {
+exports.indexCmrCollection = async (collection, g) => {
```

Comment:
The collection update event looks something like:
```
{
  "action": "concept-update",
  "concept-id": "C1200371048-DEMO_PROV",
  "revision-id": 10
}
```
We should properly handle at least the action type in this PR. We can write a separate ticket to handle the revision-id.